custom_download_strategy.rb

```diff
@@ -103,6 +103,6 @@ def resolve_asset_id
 
   def fetch_release_metadata
     release_url = "https://api.github.com/repos/#{@owner}/#{@repo}/releases/tags/#{@tag}"
-    GitHub.open_rest(release_url)
+    GitHub.open_api(release_url)
```

Comment:
It's actually `open_api` that has been deprecated in favour of `open_rest`

https://github.com/Homebrew/brew/blob/master/Library/Homebrew/utils/github.rb#L16-L20

Reply:
@robjacoby the thing is there's no such a method `Github.open_rest` any more, :)

Reply:
Ah it looks like it's now namespaced `GitHub::API.open_rest`

Reply:
indeed :)

Reply:
```suggestion
    GitHub::Api.open_rest(release_url)
```